cmd/init-repo/main.go

```diff
@@ -0,0 +1,212 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/golang/glog"
+	yaml "gopkg.in/yaml.v2"
+
+	"strings"
+
+	"k8s.io/publishing-bot/cmd/publishing-bot/config"
+)
+
+const (
+	depCommit   = "7c44971bbb9f0ed87db40b601f2d9fe4dffb750d"
+	godepCommit = "tags/v79"
+)
+
+var (
+	SystemGoPath = os.Getenv("GOPATH")
```

Comment:
i'm 90% sure there is some special package in Go that has this (like os.GOPATH or something ;-) but googling it is hell.